rowboat/util/input.py

```diff
@@ -15,6 +15,9 @@ def parse_duration(raw, source=None, negative=False):
     if not raw:
         raise CommandError('Invalid duration')
 
+    if source == None:
```

Comment:
But the other code won't work then, parse_duration(duration, timedelta()) relies on this. I could rewrite the other code to substract datetime.utcnow() instead if you want.